pkg/cmd/backuprestore/backupserver.go

```diff
@@ -124,6 +124,7 @@ func (b *backupServer) Run(ctx context.Context) error {
 		bck := backupRunnerImpl{}
 		err := b.scheduleBackup(cCtx, bck)
 		if err != nil {
+			klog.Infof("error running etcd backup: %v", err)
```

Comment:
@tjungblu  I have added this here before returning the error 

iiuc, the error will keep the pod CLOB, which will bring the whole etcd pod into `not-ready` status 


would disabling the container upon consecutive failure a solution ? or shall we silent the error and go `degraded` with CEO ? 

cc  @hasbro17  @dusk125  @vrutkovs

Reply:
imo CEO should have a separate condition for "failed to schedule a backup". I don't think it should be influencing Degraded, as it doesn't mean the operator can't recover or continue to function

Reply:
@vrutkovs shall I return `error` here, or how should I react to this situation ? 

the error will keep the Pod `NotReady` which is the same issue for all etcd-pods, and then the whole etcd-cluster is down

Reply:
We certainly need to keep this pod from crashing no matter what

Reply:
will this transpire as the termination message?

Reply:
I don't exactly see how that facebook post is helping, but please add a test to verify this behavior

Reply:
I am so sorry ... terribly sorry